packages/core-data/src/queried-data/reducer.js

```diff
@@ -138,6 +140,37 @@ const queries = flowRight( [
 	);
 } );
 
+/**
+ * Reducer tracking queries state.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Next state.
+ */
+const queries = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case 'RECEIVE_ITEMS':
+			return receiveQueries( state, action );
+		case 'REMOVE_ITEMS':
+			const newState = { ...state };
+			const removedItems = action.items.reduce( ( result, item ) => {
+				result[ item.toString() ] = true;
```

Comment:
What do you expect `item.toString()` to be here?

Reply:
Bah, it's just an artifact of various testing.